### ISSUE TITLE
Exporting with media is broken

### ISSUE THREAD

**Issue by andishehkis** [ankidroid/Anki-Android#3636]
hi
i used the latest version  AnkiDroid-2.5alpha54
the issue is about exporting a deck from ankidroid when this deck includes media file.
at first you import a deck which includes media file into ankidroid
then try to export this deck. in the export page please check the "include media"
unfortunately the media doesn't includes in exported file(the size of exported file is much less than the base file because it doesn't include media files).
this problem also occurred in the previous versions like AnkiDroid-2.4.3. in this version the size of exported apkg file shows that the media was exported but when I reloaded this file into desktop version this problem occurred.

```
Import failed.
Traceback (most recent call last):
  File "C:\cygwin\home\dae\win\build\pyi.win32\anki\outPYZ1.pyz/aqt.importing", line 327, in importFile
  File "C:\cygwin\home\dae\win\build\pyi.win32\anki\outPYZ1.pyz/anki.importing.apkg", line 25, in run
  File "C:\cygwin\home\dae\win\build\pyi.win32\anki\outPYZ1.pyz/anki.importing.anki2", line 30, in run
  File "C:\cygwin\home\dae\win\build\pyi.win32\anki\outPYZ1.pyz/anki.importing.anki2", line 45, in _import
  File "C:\cygwin\home\dae\win\build\pyi.win32\anki\outPYZ1.pyz/anki.importing.anki2", line 86, in _importNotes
  File "C:\cygwin\home\dae\win\build\pyi.win32\anki\outPYZ1.pyz/anki.importing.anki2", line 392, in _mungeMedia
  File "C:\cygwin\home\dae\win\build\pyi.win32\anki\outPYZ1.pyz/anki.media", line 216, in transformNames
  File "C:\cygwin\home\dae\win\build\pyi.win32\anki\outPYZ1.pyz/re", line 151, in sub
  File "C:\cygwin\home\dae\win\build\pyi.win32\anki\outPYZ1.pyz/anki.importing.anki2", line 372, in repl
  File "C:\cygwin\home\dae\win\build\pyi.win32\anki\outPYZ1.pyz/anki.importing.apkg", line 37, in _srcMediaData
  File "C:\cygwin\home\dae\win\build\pyi.win32\anki\outPYZ1.pyz/zipfile", line 869, in read
  File "C:\cygwin\home\dae\win\build\pyi.win32\anki\outPYZ1.pyz/zipfile", line 892, in open
  File "C:\cygwin\home\dae\win\build\pyi.win32\anki\outPYZ1.pyz/zipfile", line 859, in getinfo
KeyError: "There is no item named u'176' in the archive"
```

**Comment by george84** [comment]
I had almost exactly same error in 2.4.3. Only difference last line in error log says:
KeyError: "There is no item named u'11' in the archive"

Additional info:
When trying export of complete collection (with media) I got error message "error exporting collection" on Android. But when exporting every single deck individually the export succeeded for every deck, but later there was above import error on one specific deck.

I later found out that I had a card which referenced a missing file. After correcting this the import succeeded. (But I a still not able to do a full export)

**Comment by hssm** [comment]
Related to https://github.com/ankidroid/Anki-Android/issues/3800

**Comment by txx18** [comment]
I had almost exactly same error in PC Anki 2.1.0. I exported the deck on Ankidroid one year ago . 
Now I cannot import it neither on PC nor Ankidroid. 
 PC2.1.0 anki says:KeyError: **"There is no item named '27' in the archive"**
Ankidroid2.8.3 can show me the name of deck to import, but when I push confirm button , it failed can logged nothing.
It is a big deck with media and it is important to me. Can I open it again? Please help me.

**Comment by mikehardy** [comment]
2.9.4 might be more tolerant of the errors on import. You could try it.

**Comment by mikehardy** [comment]
I'm nearly positive this is fixed+closed, but if you retest on AnkiDroid 2.9.4+ and still have problems please let us know
